topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md

```diff
@@ -0,0 +1,412 @@
+>
+>Using this formula, we can calculate that a quality score of 40 means only 0.00010 probability of an error!
+>
+> Learn more from the [Quality Control Tutorial FAQs](https://training.galaxyproject.org/training-material/topics/sequence-analysis/tutorials/quality-control/faqs/).
+>
+{: .details}
+
+# Exercise Three: Alignment
+
+Given that our data has passed some quality checks, we will try to align the data to the reference genome. In this case it is simple, a viral genome. A human sequencing project will generate much larger data sets. There are many aligners, but we will start off looking at a simple aligner BWA-MEM. This example uses paired data.
+
+We will use our two SARS data files, which are ready for alignment.
+
+- VA_sample_forward_reads.fastq
+- VA_sample_reverse_reads.fastq
+
+> <hands-on-title>Align to the Reference Dataset</hands-on-title>
+>
+> 1. Go to GENOMICS ANALYSIS and expand the Mapping menu. Select {% tool [Map with BWA-MEM](toolshed.g2.bx.psu.edu/repos/devteam/bwa/bwa_mem/0.7.17.2) %}. This program will align your reads to your SARS reference genome. Some of our reads are >100 base pairs so we will use the MEM option. 
+>    - *"Will you select a reference genome from your history or use a built-in index?"*: Use a genome from history and build index.
+>    - {% icon param-file %} *"Use the following dataset as the reference sequence"*: `SARS-CoV-2_reference_genome.fasta`
+>    - *"Single or Paired-end reads"*: `Paired`
+>    - *"Select first set of reads"*: `VA_sample_forward_reads.fastq`
+>    - *"Select second set of reads"*: `VA_sample_reverse_reads.fastq`
+> 2. Run the tool.
+>
+{: .hands_on}
+
+The output file is a `BAM` file, which lists where each read aligns to the reference genome and whether there are any differences. You can click the eye button to preview the results, but the results are not easy to interpret visually (much like the `fastq` files). Instead you will use a genome viewer in the next step.
+
+> <question-title></question-title>
+>
+> 1. What is alignment software (for example, BWA-MEM) actually doing?
+> 2. Here we are using paired fastq (“paired end”) data. What is an advantage of using paired data?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. Alignment tools figure out the optimal positioning of the reads next to the reference genome to minimize mismatches and gaps. 
+> > 2. Paired data is made up of fragments that are read twice (forward and reverse). Paired data improves data accuracy.
+> >
+> {: .solution}
+{: .question}
+
+# Viewing aligned data
+
+We have aligned our data but it is currently a table of where the reads align. This is hard to read, so we will use [JBrowse](https://jbrowse.org/jbrowse1.html) to view the data.
+
+> <hands-on-title>Visualize Reference Data</hands-on-title>
+>
+> 1. Scroll down in the Tools menu to STATISTICS AND VISUALIZATION. Under "Graph/Display Data", select {% tool [JBrowse](https://toolshed.g2.bx.psu.edu/view/iuc/jbrowse/a6e57ff585c0) %}.
```

Comment:
This also should have failed linting, I really don't understand :| 

This isn't the right format for a tool link, it should look like 
```suggestion
> 1. Scroll down in the Tools menu to STATISTICS AND VISUALIZATION. Under "Graph/Display Data", select {% tool [JBrowse](toolshed.g2.bx.psu.edu/repos/iuc/jbrowse/jbrowse/1.16.9+galaxy0) %}.
```